src/fides/config/job_settings.py

```diff
@@ -0,0 +1,35 @@
+from typing import Optional
+
+import validators
+from pydantic import Field, validator
+
+from .fides_settings import FidesSettings
+
+ENV_PREFIX = "FIDES__JOBS__"
+
+
+class JobSettings(FidesSettings):
+    """Configuration settings for job scheduling."""
+
+    # System change digest
+    system_change_webhook_url: Optional[str] = Field(
```

Comment:
You might be able to set an optional AnyUrl type `from pydantic import AnyUrl` to avoid raising the ValueError below